src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-policyset-definition/main.combined.bicep

```diff
@@ -0,0 +1,44 @@
+resource policySetDefinition 'Microsoft.Authorization/policySetDefinitions@2020-09-01' = {
+//@[89:1023) [BCP135 (Error)] Scope "resourceGroup" is not valid for this resource type. Permitted scopes: "managementGroup", "subscription". (CodeDescription: none) |{\r\n  name: 'name'\r\n  properties: {\r\n    displayName: 'displayName'\r\n    policyType: 'Custom'\r\n    description: 'description'\r\n    metadata: {\r\n      version: '0.1.0'\r\n      category: 'category'\r\n      source: 'source'\r\n    }\r\n    parameters: {\r\n      parameterName: {\r\n        type: 'String'\r\n        metadata: {\r\n          displayName: 'displayName'\r\n          description: 'description'\r\n        }\r\n      }\r\n    }\r\n    policyDefinitions: [\r\n      {\r\n        policyDefinitionId: 'policyDefinitionId'\r\n        policyDefinitionReferenceId: 'policyDefinitionReferenceId'\r\n        parameters: {\r\n          parameterName: {\r\n            value: 'value'\r\n          }\r\n        }\r\n      }\r\n      {\r\n        policyDefinitionId: 'policyDefinitionId'\r\n        policyDefinitionReferenceId: 'policyDefinitionReferenceId'\r\n        parameters: {\r\n          parameterName: {\r\n            value: 'value'\r\n          }\r\n        }\r\n      }\r\n    ]\r\n  }\r\n}|
```

Comment:
Fix me

Reply:
TargetScope will need to be added.

Reply:
** Only include TargetScope in main.combined.bicep not snippet template itself.

Reply:
thanks, added! bb6d5948d55f8898230ba8fc208ad7879037ba99